samples/benchmarks/coremark/README.rst

```diff
@@ -157,14 +145,14 @@ SB_CONFIG_APP_CPUPPR_RUN - Enable execution for the PPR core
    Enable the benchmark execution also for the PPR core for targets with the nRF54H20 SoCs.
 
 .. note::
-   PPR code is run from MRAM with the execution in place (XIP) method.
-   You must use the ``nordic-ppr-xip`` snippet for the application core to be able to boot the PPR core.
-   Use the build argument ``coremark_SNIPPET=nordic-ppr-xip``.
+   PPR code is run from RAM.
```

Comment:
same as above - RAM30 (TCM for PPR).

Reply:
We did not provide such information for MRAM in the XIP mode (the exact MRAM block number that is used for the specific core). Getting into the platform-specific details here would make this documentation harder to maintain

Reply:
I can provide information regarding the TCM if you believe that using it for PPR will be standard approach for all platforms